README.md

```diff
@@ -0,0 +1,491 @@
+
+## Update User
+**Needs to be logged in/auth token**
+
+Update an existing user by making a **PUT** request to the following endpoint:
+
+```http
+PUT /:orgId/:id
+```
+
+**Request Body:**
+
+```json
+{
+  "firstName": "Updated John"
+}
+```
+
+**Response:**
+
+- **Status Code:** `200 OK`
+- **Response Body:** The updated user object.
+
+## Delete User
+**Needs to be logged in/auth token**
+
+Delete a specific user by their ID for a given organization by making a **DELETE** request to the following endpoint:
+
+```http
+DELETE /:orgId/:id
+```
+
+**Response:**
+
+- **Status Code:** `204 No Content`
+
+## Forgot Password
+Initiate the forgot password process by making a **POST** request to the following endpoint:
+
+```http
+POST /:orgId/forgot-password
+```
+
+**Request Body:**
+
+```json
+{
+  "email": "john.doe@example.com"
+}
+```
+
+**Response:**
+
+- **Status Code:** `200 OK`
+- **Response Body:** An object with a message indicating that the email has been sent.
+
+# Error Handling Documentation
+
+## Overview
+
+The error handling module in this application is designed to manage and respond to errors that may occur during the execution of API endpoints. It includes middleware functions for logging errors, responding with appropriate error messages, and handling invalid paths.
+
+## Error Logger Middleware
+
+The `errorLogger` middleware function logs error messages to the console.
+
+### Usage
+
+```javascript
+import { errorLogger } from 'path-to-error-handler';
+
+// ... Other middleware and route handling
+
+app.use(errorLogger);
+```
+
+## Error Responder Middleware
+
+The ```errorResponder``` middleware function sends JSON-formatted error responses.
+
+### Usage
+
+```javascript
+import { errorResponder } from 'path-to-error-handler';
+
+// ... Other middleware and route handling
+
+app.use(errorResponder);
+```
+
+### Custom Errors
+
+Custom errors, such as ```BaseError```, are supported. They can be thrown with a specific status code and optional error details.
+
+```javascript
+import { BaseError } from 'path-to-error-handler';
+
+throw new BaseError('Custom error message', 404, { additionalInfo: 'details' });
+```
+
+## Invalid Path Handler
+
+The ```invalidPathHandler``` middleware function handles undefined paths, returning a 404 error.
+
+### Usage
+
+```javascript
+import { invalidPathHandler } from 'path-to-error-handler';
+
+// ... Other middleware and route handling
+
+app.use(invalidPathHandler);
+```
+
+## Password Validation
+
+A list of common passwords is available for use in password validation.
+
+### Usage
+
+```javascript
+import { commonPasswords100 } from 'path-to-error-handler';
+
+// Example of password validation
+const userPassword = 'password123';
+if (commonPasswords100.includes(userPassword)) {
+  // Password is common, handle accordingly
+}
+```
+
+### Common Passwords List
+
+The ```commonPasswords100``` array contains commonly used passwords and can be used for password strength validation.
```

Comment:
Why include this? Seems a bit GPT-like to me.

Reply:
it is :)